lib/listcoef.gd

```diff
@@ -101,26 +101,53 @@ DeclareOperation(
 #O  MultRowVector( <list>, <mul> )
 ##
 ##  <#GAPDoc Label="MultRowVector">
+##  <#GAPDoc Label="MultRowVectorLeft">
+##  <#GAPDoc Label="MultRowVectorRight">
```

Comment:
Why are these called Mult*Row*VectorLeft/Right? In the new scheme, they should be `MultVectorLeft/Right`, no?

Reply:
Last week, you mentioned wanting to rename them in a second step later on, but I don't understand why -- why would we introduce a new name, only to then rename it immediately after?

Reply:
Also note that the above is invalid syntax, and cause `make doc` to fail.